packages/networking/lib/agent.ts

```diff
@@ -251,19 +269,11 @@ class HttpsAgent extends https.Agent {
     proxySocket.once('error', onError)
     proxySocket.once('data', onData)
 
-    let connectReq = `CONNECT ${hostname}:${port} HTTP/1.1\r\n`
+    const connectReq = _buildConnectReqHead(hostname, port, proxy)
 
-    connectReq += `Host: ${hostname}:${port}\r\n`
-
-    if (proxy.auth) {
-      connectReq += `Proxy-Authorization: basic ${Buffer.from(proxy.auth).toString('base64')}\r\n`
-    }
-
-    connectReq += '\r\n'
     proxySocket.write(connectReq)
   }
 }
 
-module.exports = new CombinedAgent()
-
-module.exports.CombinedAgent = CombinedAgent
+// @ts-ignore
+module.exports = Object.assign(new CombinedAgent(), module.exports)
```

Comment:
I think at this point rather than shoehorn a pseudo default export (which is the combined agent instance) and attaching everything to that agent instance you should just export the `agent` instance as a separate property and keep the others where they're at.

```js
// call it a day
export const agent = new CombinedAgent()

// and import it in normally
import { agent, CombinedAgent, isRequestHttps, buildConnectReqHead } from '@packages/network'
```